tests/test_train.py

```diff
@@ -76,7 +76,7 @@ def test_create_scibert_svm():
 def test_create_bert():
     model = create_model('bert')
     assert isinstance(model, BertClassifier)
-    assert model.pretrained == 'bert'
+    assert model.pretrained == 'bert-base-uncased'
```

Comment:
why uncased? smaller?

Reply:
I was always using the uncased model, have you been using the cased on?